module-services/service-cellular/src/VolteHandler.hpp

```diff
@@ -103,6 +114,47 @@ namespace cellular::service
             return isFirstRunAfterSwitch;
         }
 
+        struct NetworkServiceDomain
+        {
+            enum class Type
+            {
+                SmsViaIms,
+                SmsViaSgs,
+                Voice
+            };
+            explicit NetworkServiceDomain(Type type) : type_{type}
+            {
+                switch (type) {
+                case Type::SmsViaSgs:
+                    domain_     = "sms";
+                    preference_ = 0x00;
+                    return;
+                case Type::SmsViaIms:
+                    domain_     = "sms";
+                    preference_ = 0x01;
+                    return;
+                case Type::Voice:
+                    domain_     = "voice";
+                    preference_ = 0x03;
+                    return;
+                default:
+                    throw std::logic_error("unimplemented network service domain: " +
+                                           std::string(magic_enum::enum_name(type)));
+                }
+            }
+
+            operator std::string()
+            {
+                return "\"/nv/item_files/modem/mmode/" + domain_ + "_domain_pref\"," +
+                       utils::byteToHex<std::uint8_t>(preference_);
+            }
+
+          private:
+            std::string domain_;
+            Type type_;
+            std::uint8_t preference_;
```

Comment:
Are there guidelines regarding the names of member variables? I can see many different styles in the code base but haven't seen this one yet. We should probably talk about this and define a clear standard.

Reply:
That's right. This one probably already exists in some pieces of code written by me. It's clearer than `mSomething` and the underscore is trailing because a leading one can mean an UB in some cases.

Reply:
@awulkiew So, for now I'm closing this conversation - please arrange a meeting or a Slack discussion as this topic is recurring sometimes.